modules/admin_manual/pages/configuration/user/user_management.adoc

```diff
@@ -242,3 +242,64 @@ against quotas.
 
 When a user creates a public link share via URL, and allows uploads, any
 uploaded files count against that user’s quota.
+
+== Moving the User Home
+
+Usually, a users home directory is located in the `data/` directory which location is defined in `config.php` and defaults to `/var/www/owncloud/data`. While in smaller installations, the data directory mounts to a physical drive of the server and in bigger installations the data directory is mounted via NFS, this can be suboptimal when having very big installations with thousands of users and/or users having very big space consumtions. This can result in having high load on a single mount point.
+
+owncloud offers moving the home directory location of single users from the default location to another path outside the data directory to distribute the load on the mount points. You can also move back a users home to the location defined in config.php.
+
+NOTE: As a general advice, moving a users home is the last step after all optimisations of the mount point have been set. Carefully monitor over a period of time the changes made before moving a users home.
+
+NOTE: Consider that these new home mount points also need to be part of a xref:maintenance/backup_and_restore/backup.adoc[backup] and xref:maintenance/backup_and_restore/restore.adoc[restore] plan, if one exists.
+
+=== Steps to Move a Users Home
+
+For details of the occ user commands used below, see the xref:configuration/server/occ_command.adoc#user-commands[User Commands] section of the occ command reference. The examples use the user ID `lisa` and the mount for the new home is `/mnt/newhome_1`. 
+
+. To report a users home, use the following command:
++
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} user:list lisa -a home
+  - lisa: /var/www/owncloud/data/lisa
+----
++
+Here you can see, that the home of user `lisa` is located in `/var/www/owncloud/data/lisa`
+
+. Prepare new mounts *in advance* for one or more users:
++
+Use the OS methods to create one or more new mount points for users home directories not located as defined in config.php. You can create as many new homes as required containaing as many new user homes as wanted - there is no ownCloud limitation. The mounts must be fully accessible by the webserver user (usually www-data).
+
+. Move the users home:
++
+To move a users home, the target folder *must* be empty !
```

Comment:
I think you mean that the folder `/mnt/newhome_1` must not have a folder `lisa` (or at least `/mnt/newhome_1/lisa` must be empty).

`/mnt/newhome_1` might already have home folders for `phil` `martin` etc. This could be clarified somewhere - that the folder `/mnt/newhome_1` is for storing one or two or ... many user folders.

Reply:
ah, you are right, this needs a text fix

Reply:
fixed, squashed the commits